packages/frontend/src/components/common/modal/Modal.js

```diff
@@ -45,7 +45,16 @@ function Modal({
 
     },[]);
 
-    useEffect(() => { setFadeType('out'); }, [isOpen]);
+    useEffect(() => { 
+        if (isOpen) {
+            setFadeType('out');
```

Comment:
Can this go at the top of the function and then drop the `else`?

Reply:
💯